pandas/tests/io/excel/test_readers.py

```diff
@@ -1437,7 +1437,9 @@ def test_deprecate_bytes_input(self, engine, read_ext):
             "byte string, wrap it in a `BytesIO` object."
         )
 
-        with tm.assert_produces_warning(FutureWarning, match=msg):
+        with tm.assert_produces_warning(
+            FutureWarning, match=msg, raise_on_extra_warnings=False
```

Comment:
What's the extra warning here?

Reply:
a DeprecationWarning that originates in the excel libraries